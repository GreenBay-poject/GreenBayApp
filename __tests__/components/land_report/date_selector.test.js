import React from 'react';
import renderer from 'react-test-renderer';

import Date_Selector from '../../../components/land_report/date_selector'

describe('<Date_Selector />', () => {
    it('Sample Date_Selector', () => {
        const tree = renderer.create(<Date_Selector dates={['2021-08-11']} setdates={null} latitude={80.34} longitude={23.54} date={'2021-08-11'} setdate={null} setstep={null} />).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree.type).toBe('View');
        expect(tree.children[0].type).toBe("View");
        expect(tree.children[1].type).toBe("View");
        expect(tree.children[2].type).toBe("RCTScrollView");
        //console.log(tree)
    });
});