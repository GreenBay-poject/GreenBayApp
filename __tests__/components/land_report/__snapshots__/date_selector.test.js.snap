// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Date_Selector /> Sample Date_Selector 1`] = `
<View>
  <View
    style={
      Object {
        "marginBottom": 0,
        "marginTop": 0,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "#153d37",
          "borderRadius": 10,
          "opacity": 1,
          "padding": 10,
          "width": "95%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 14,
          }
        }
      >
        FETCH IMAGE  
        2021-08-11
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "marginBottom": 2,
        "marginTop": 2,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "#153d37",
          "borderRadius": 10,
          "opacity": 1,
          "padding": 10,
          "width": "95%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 14,
          }
        }
      >
        BACK
      </Text>
    </View>
  </View>
  <RCTScrollView
    style={
      Object {
        "height": "80%",
        "marginTop": 20,
      }
    }
  >
    <View>
      2021-08-11
    </View>
  </RCTScrollView>
</View>
`;
